google/cloud/forseti/services/inventory/crawler.py

```diff
@@ -112,6 +116,12 @@ def visit(self, resource):
             Exception: Reraises any exception.
         """
 
+        attrs = {
```

Comment:
it's not clear, but curious how these `attrs` are used.

Reply:
These attributes will be added to the current span, if any.